collect/collect.go

```diff
@@ -100,17 +209,32 @@ func (c *InventoryCollection) Collect() {
 func (c *InventoryCollection) Upload() error {
 	log.Info().Msg("uploading inventory")
 
-	payload, err := json.Marshal(c.Inventory)
+	var (
+		payload     []byte
+		err         error
+		contentType string = "application/json; charset=UTF-8"
+	)
+
+	payload, err = json.Marshal(c.Inventory)
 	if err != nil {
 		return errors.Wrap(err, "marshaling inventory")
 	}
 
+	if c.AuthEnabled {
+		jwt, err := c.Signer.Sign(payload)
```

Comment:
strictly this is the `jws` - we have no jwt the payload is what normally (in OpenID Connect at least) would have been a jwt.

Reply:
fixed